requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy
 accelerate>=0.20.3
 packaging
-transformers>=4.34.1
+transformers>=4.34.1,<4.38.0
```

Comment:
the reason you are seeing this error is because `TrainingArguments` is not initialized properly. In the code, we inherit from the dataclass, but we did not call `.__post_init__.` This PR #53 fixes this issue.